src/main/java/com/flightstats/hub/webhook/Webhook.java

```diff
@@ -294,6 +324,9 @@ public boolean equals(Object o) {
         final Object this$name = this.getName();
         final Object other$name = other.getName();
         if (this$name == null ? other$name != null : !this$name.equals(other$name)) return false;
+        final Object this$tag = this.getTag();
```

Comment:
What do you think about reverting to lombok?

Reply:
created an issue to relombok

Reply:
#938